docs/using_mosaics_api.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
Great notebook! Can we put this in https://github.com/NASA-IMPACT/titiler/tree/master/docs/examples/notebooks/ just for consistency?

Reply:
done